recipes/mimosa/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+mkdir cmake_build && cd cmake_build
+cmake -DCMAKE_PREFIX_PATH=${PREFIX} -DCMAKE_INSTALL_PREFIX=${PREFIX} -DUSE_CPLEX=0 ..
```

Comment:
```suggestion
cmake ${CMAKE_ARHS} -DUSE_CPLEX=0 ..
```

Reply:
Just wondering if there is a typo in your suggestion: `CMAKE_ARHS` or `CMAKE_ARGS` ?
(overall thanks a lot for the review, first-time contributor to conda, and not a very skilled user of CMake...)